index.yaml

```diff
@@ -178,10 +178,28 @@ entries:
     urls:
     - https://apps.okteto.com/redis/redis-9.2.2.tgz
     version: 9.2.2
+  tensorflow-notebook:
+  - apiVersion: v1
+    appVersion: 1.6.0
+    created: "2020-02-04T17:22:28.330826+01:00"
+    description: A Helm chart for tensorflow notebook and tensorboard
+    digest: 43139e567f2b489b47419c6869a1726d6746c567c3c4368ca0b424708d13ad50
+    home: https://www.tensorflow.org
+    icon: https://apps.okteto.com/tensorflow-notebook
```

Comment:
this needs to be an image. @rlamana do we need to do any treatment to the image or will the default image just work?

Reply:
I had prepared it to put an image. I fix it.